stdlib/hashtbl.mli

```diff
@@ -216,6 +216,20 @@ val stats : ('a, 'b) t -> statistics
    buckets by size.
    @since 4.00.0 *)
 
+val of_list : ('a * 'b) list -> ('a, 'b) t
+(** [Hashtbl.of_list l] is a table built from the given bindings. The bindings
+    are added in the same order they appear in [l], using {!add}.
+
+    @since 4.09 *)
+
+val to_list : ('a, 'b) t -> ('a * 'b) list
+(** [Hashtbl.to_list h] is the list of bindings in [h]. The order in which the
+    bindings appear in the list is unspecified.  However, if the table contains
+    several bindings for the same key, they appear in order of introduction,
+    that is, the most recent binding appears last.
+
+    @since 4.09 *)
+
```

Comment:
In terms of documentation, could these go with the Seq functions? (ie as iterators)

Reply:
Yes, I moved them under `Iterators`.